pyproject.toml

```diff
@@ -46,6 +46,7 @@ dask = {extras = ["dataframe"], version = ">= 2023.4.1"}
 importlib-resources = { version = ">= 1.3", python = "<3.9" }
 jsonschema = ">= 4.18"
 pyarrow = ">= 11.0.0"
+graphviz = ">= 0.20.1"
```

Comment:
same comment here as above, maybe for testing you can just have it in the `requirements.txt` for now